### PR TITLE
Fix typos in CSRNG enumerated values

### PR DESCRIPTION
**Pull request by rivos-eblot**
For now: blind commit, as I'm unable to build the master branch on our machine. To be edited...

### CODE REVIEW

**Review by engdoreis — APPROVED** [review]
Thanks for these fixes.

**Review by marnovandermaas — COMMENTED** [review]
For the first two commits, someone should build the DIFs to check the changes. For the last commit, someone should run a test that uses `entropy_csrng_uninstantiate`.

If you would like me to do this testing, please let me know the tests you would like to run.

**Comment by rivos-eblot** [comment]
> For the last commit, someone should run a test that uses `entropy_csrng_uninstantiate`.
> If you would like me to do this testing, please let me know the tests you would like to run.

`entropy_test` uses this API if you can give it a try, thanks. See also #19293.

File: sw/device/lib/crypto/drivers/entropy.c [hunk 1]
```diff
@@ -596,7 +596,7 @@ status_t entropy_csrng_generate(const entropy_seed_material_t *seed_material,
 status_t entropy_csrng_uninstantiate(void) {
   return csrng_send_app_cmd(kBaseCsrng + CSRNG_CMD_REQ_REG_OFFSET,
                             (entropy_csrng_cmd_t){
-                                .id = kEntropyDrbgOpUpdate,
+                                .id = kEntropyDrbgOpUninstantiate,
```
[the remark this answers is not in the document]
Comment (engdoreis) [reply sw/device/lib/crypto/drivers/entropy.c:599]: Good point, this function is used on the manufacturing tests, so I think we can trust CI? @timothytrippel would know more.

**Review by moidx — APPROVED** [review]

**Pull request merged**